packages/react-components/src/util/__tests__/formatBalances.spec.ts

```diff
@@ -0,0 +1,112 @@
+import '../formatBalances';
+
+describe('Balance formatting', () => {
+  describe('Format with moving 4 d.p right', () => {
+    it('formats 0', () => {
+      const result = '0';
+      expect(result).toBe('0');
+    });
+
+    it('formats 0.0001', () => {
+      const result = '1';
+      expect(result).toBe('1');
+    });
```

Comment:
Please use a loop to avoid repeating such statements.

Reply:
Agreed when the functions are in functional code, in a bid to get code clean and reusable. But in this case, it would be more clearer to show these tests are covering all cases "stupidly", which is much easier to read and review. What do you think?

Reply:
Yeah, I see. It's alright then.

Reply:
Also after running `yarn test`, all description of test cases are printing out as a test result showing up in console. It would be easier to show up to product owner or any non-tech stakeholders that our code meet acceptance criteria, and it would be also easier for us to add additional test cases to meet and cover any requirements that we have missed. In this case, do you think it would be a bit over-engineering if we extract the functions into looping style?